chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -597,8 +597,8 @@ impl PeerManagerActor {
                             anyhow::Ok(())
                         }.await;
 
-                        if result.is_err() {
-                            tracing::info!(target:"network", ?result, "failed to connect to {peer_info}");
+                        if let Err(ref err) = result {
+                            tracing::info!(target: "network", err = %err, "failed to connect to {peer_info}");
```

Comment:
For some reason below does not compile:

```suggestion
                            tracing::info!(target: "network", %err, "failed to connect to {peer_info}");
```

Checked what rust-analyzed is reporting and it expands into `Some(&err as &Value)` instead of `Some(&display(&err)as &Value)`